### DISCUSSION TITLE
Secondary database for PHP unit testing in WordPress

### DISCUSSION THREAD

**Discussion by damonsharp** [docksal/docksal#1721] [Ideas]
When doing PHP unit testing in WordPress a secondary testing db is spun up during setup. Is it possible to do this inside Docksal? From my understanding it's one `default` db per project. If not possible currently, any thoughts to allowing additional db creation for this purpose?

**Comment by sean-e-dietrich** [comment]
hi @damonsharp yes you can run `fin db create [dbname]` example `fin db create migrate` and now use the following credentials to access it.

```
username: user
password: user
hostname: db
dbname: migrate
```

when you import a database you can run `fin db import --db=migrate file.db.sql`

**Reply by damonsharp** [reply]
THANKS!
